app/main.py

```diff
@@ -1 +1,21 @@
-# write your code here
+from app.cafe import Cafe
+
+from app.errors import (
+    NotVaccinatedError,
+    OutdatedVaccineError,
+    NotWearingMaskError
+)
+
+
+def go_to_cafe(friends: list, cafe: Cafe) -> str:
+    masks_to_buy = 0
+    for friend in friends:
+        try:
+            cafe.visit_cafe(visitor=friend)
+        except (NotVaccinatedError, OutdatedVaccineError):
```

Comment:
You must except here "Vaccine Error".  NotVaccinatedError, OutdatedVaccineError Inherited from VaccineError.